File: 2d_array/cpp/no2738.cpp
#include <iostream>
#include <deque>

using namespace std;

int main() {
    int n, m;
    cin >> n >> m;

    deque<deque<int>> matrix_a(n, deque<int>(m));
    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < m; ++j) {
            cin >> matrix_a[i][j];
        }
    }

    deque<deque<int>> matrix_b(n, deque<int>(m));
        for (int i = 0; i < n; ++i) {
            for (int j = 0; j < m; ++j) {
                cin >> matrix_b[i][j];
        }
    }

    for (int i = 0; i < n; ++i) {
        for (int j = 0; j < m; ++j) {
            cout << matrix_a[i][j] + matrix_b[i][j] << " ";
        }
        cout << endl;
    }
    
    return 0;
}

File: 2d_array/python/no2738.py
n, m = map(int, input().split())

matrix_a = []
for _ in range(n):
    row = list(map(int, input().split()))
    matrix_a.append(row)

matrix_b = []
for _ in range(n):
    row = list(map(int, input().split()))
    matrix_b.append(row)

result = [
    [0] * m for _ in range(n)
]

for i in range(n):
    for j in range(m):
        result[i][j] += matrix_a[i][j] + matrix_b[i][j]

for row in result:
    print(' '.join(map(str, row)))